src/main/java/ProjektiPizza/demo/security/JwtGenerator.java

```diff
@@ -0,0 +1,22 @@
+package ProjektiPizza.demo.security;
+
+import ProjektiPizza.demo.entity.User;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JwtGenerator {
+
+    public String generate(User user) {
+        Claims claims = Jwts.claims()
+                .setSubject(user.getUsername());
+        claims.setSubject(user.getPassword());
+        claims.setSubject(user.getFullName());
+        return Jwts.builder()
+                .setClaims(claims)
+                .signWith(SignatureAlgorithm.HS512, "youtube")
```

Comment:
That constants class could be used here for example without duplicating the string "youtube" here

Reply:
you're right.